RocketChat/Rocket.Chat.ReactNative
# Chore: Upgrade react-native-notifications to 4.2.4

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Push notifications
- Push notifications of encrypted messages
- Push notification data privacy
- Tap on push notifications with the app **closed** and it should redirect properly (room and thread)
- Tap on push notifications with the app **in background** and it should redirect properly (room and thread)

## Screenshots

## Types of changes




## Checklist



## Further comments
